### DISCUSSION TITLE
Support for HERO11 Black Mini

### DISCUSSION THREAD

**Discussion by dnewman-gpsw** [gopro/labs#86] [General]
Yes, this is planned, hopeful for on or near it launch date.  It should have the same features of HERO11 Labs, although the lack of the rear-LCD will limit or change some on the user feedback.  The cameras small display will be enabled for Labs messaging.

**Comment by RikshaDriver** [comment]
G'day Dave,

Will the Mini also support LTC via the 3.5mm USB module?

**Reply by KonradIT** [reply]
Hero11 mini doesn't support audio in via Pro 3.5mm audio adapter (according to my review guide), so it won't support LTC sync.

**Reply by RikshaDriver** [reply]
Were you able to test it out? That's a shame if it's the case. I assume this is currently a firmware limitation.

The Mini would have been an ideal use case for LTC sync via the module in multi-shoot scenarios.

**Reply by KonradIT** [reply]
I don't have a Mini with me, but that's what it said on the review guide.

**Reply by dnewman-gpsw** [reply]
I just tried it, does not work. For Mini you will have to use QR Code timecodes.
